pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -14,34 +13,79 @@ This section describes how to upgrade your Kommander Management cluster and all
 
 ## Prerequisites
 
-- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- [Download][download_binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.
+- Download the following air-gapped bundles: [images bundle][images_bundle], [charts bundle][charts_bundle] and [Catalog Application charts bundle][cat_apps_bundle]. **Skip this if you are not running in an air-gapped environment**.  
+
+### Load the Docker images into your Docker registry (skip this section if you are running in an air-gapped environment)
+
+1.  Download the image bundle file:
+
+    ```bash
+    wget "https://downloads.mesosphere.com/kommander/airgapped/v2.2.0/kommander_image_bundle_v2.2.0_linux_amd64.tar.gz" -O kommander-image-bundle.tar.gz
+    ```
+
+1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+
+1.  Ensure you set the `REGISTRY_URL` and `AIRGAPPED_TAR_FILE` variable appropriately, then use the following script to load the air-gapped image bundle:
```

Comment:
Do we have a Jira somewhere that automates it in the CLI? If not, I can create one. IIRC we had something similar in CLI in konvoy 1.x that was automating it.

Reply:
I don't think we can change this flow at this point in 2.2 so it would have to be done in 2.3.

Reply:
This whole section is different, no need for the script below - docs site is already updated - see https://github.com/mesosphere/dcos-docs-site/blob/main/pages/dkp/kommander/2.2/install/air-gapped/index.md#load-the-docker-images-into-your-docker-registry and use the same wording.